### PR TITLE
Appending extensions in the url files

### PR DESCRIPTION
**Pull request by joshjo**
In order to allow to import gzip files, this pull request adds a configuration section to load the gzip files.

### CODE REVIEW

**Comment by owais** [comment]
I'd rather add support to the template tags so they add a suffix to the files like

```django
    {% render_bundle 'main' 'css' suffix='.gz' %}
```

**Comment by owais** [comment]
@joshjo https://github.com/ezhome/django-webpack-loader/issues/137

**Comment by joshjo** [comment]
@owais updated pull request, the suffix now is added in the template tag.

File: webpack_loader/utils.py [hunk 1]
```diff
@@ -48,11 +49,11 @@ def get_as_tags(bundle_name, extension=None, config='DEFAULT', attrs=''):
         if chunk['name'].endswith(('.js', '.js.gz')):
             tags.append((
                 '<script type="text/javascript" src="{0}" {1}></script>'
-            ).format(chunk['url'], attrs))
+            ).format(''.join([chunk['url'], suffix]), attrs))
         elif chunk['name'].endswith(('.css', '.css.gz')):
             tags.append((
                 '<link type="text/css" href="{0}" rel="stylesheet" {1}/>'
-            ).format(chunk['url'], attrs))
+            ).format(''.join([chunk['url'], suffix]), attrs))
```
Comment (owais) [inline webpack_loader/utils.py]: Why not `href="{0}{1}" {2}".format(url, suffix, attrs)`?

**Comment by jonalxh** [comment]
Please consider to merge it or make an .gzip implementation.

**Comment by owais** [comment]
Can someone confirm if this is still a viable solution and still works with the latest master before we merge it?

**Comment by fjsj** [comment]
@rvlb could you please check this one?

**Comment by joaopslins** [comment]
FYI this was just included in the 1.3.0 version

**Pull request merged**